examples/index.md

```diff
@@ -0,0 +1,27 @@
+<!-- File generated by example-index-generator, DO NOT EDIT. -->
+# Examples
```

Comment:
Seems like `update-runtime` example was accidentally deleted in #147. Also, I think example names shouldn't use capital letters.

Reply:
done